test/device_map_dialog_test.rb

```diff
@@ -0,0 +1,69 @@
+require_relative "test_helper"
+
+require "bootloader/device_map_dialog"
+
+describe Bootloader::DeviceMapDialog do
+  # just simple tests to avoid typos as logic is not so easy to test
+  describe "#run" do
+    before do
+      Yast.import "BootStorage"
+
+      device_map = Bootloader::DeviceMap.new(
+        "/dev/sda" => "hd0",
+        "/dev/sdb" => "hd1"
+      )
+      allow(Yast::BootStorage).to receive(:DisksOrder)
+        .and_return(device_map.disks_order)
+    end
+
+    def mock_ui_events(*events)
```

Comment:
yes, make sense. I am considering that maybe we can add it to rspec helper to ui. In general maybe default setup of mock could be useful.